src/app/map-location/map-location-list/map-location-list.component.css

```diff
@@ -13,6 +13,17 @@
   grid-template-areas:
     "photo-wrap audio-list"
     "info-wrap audio-list";
+
+
+  .route-button-container {
+    margin-top: 10px;
+    text-align: center;
+  }
+
+  .route-button-container .btn {
+    background-color: #28a745; /* Zielony kolor dla przycisku */
```

Comment:
![bh187-minions](https://github.com/user-attachments/assets/549e5fd1-56bf-46be-a202-98d475882cde)


pls no comment here

Reply:
your comment is in polish comment about the code is in English @Drillllll . Decide about your origin